src/test/java/uk/gov/hmcts/juror/api/moj/report/AbstractReportTest.java

```diff
@@ -101,7 +101,7 @@ void sizeCheck() {
             assertThat(AbstractReport.CLASS_TO_JOIN.get(QPanel.panel)).hasSize(1);
             assertThat(AbstractReport.CLASS_TO_JOIN.get(QJuror.juror)).hasSize(3);
             assertThat(AbstractReport.CLASS_TO_JOIN.get(QJurorPool.jurorPool)).hasSize(1);
-            assertThat(AbstractReport.CLASS_TO_JOIN.get(QPoolRequest.poolRequest)).hasSize(1);
+            assertThat(AbstractReport.CLASS_TO_JOIN.get(QPoolRequest.poolRequest)).hasSize(2);
```

Comment:
Should also add test to validate the second CLASS_TO_JOIN predicates

Reply:
updated